flixel/tweens/FlxTween.hx

```diff
@@ -406,8 +410,61 @@ class FlxTween implements IFlxDestroyable
 		onUpdate = null;
 		onComplete = null;
 		ease = null;
+		_thens = null;
 	}
-
+	
+	/**
+	 * After this tween has finished, do this tween next
+	 * @param	method
+	 * @param	params
+	 * @return
+	 */
+	
+	public function then(Tween:FlxTween):FlxTween
+	{
+		Tween.cancelSoft();
+		if (_thens == null)
+		{
+			_thens = [];
+		}
+		_thens.push(new ThenCommand(0, Tween));
+		return this;
+	}
+	
+	/**
+	 * After this tween has finished, wait this many seconds, then do the next chained "then" command
+	 * @param	Delay The number of seconds to wait
+	 * @return
+	 */
+	public function wait(Delay:Float):FlxTween
+	{
+		if (_thens == null)
+		{
+			_thens = [];
+		}
+		_thens.push(new ThenCommand(Delay, null));
+		return this;
+	}
+	
+	/**
+	 * Add a delay and a chained tween in the same command
+	 * @param	Delay
+	 * @param	method
+	 * @param	params
+	 * @return
+	 */
+	public function waitThen(Delay:Float, Tween:FlxTween):FlxTween
```

Comment:
Is there really much of a point to this combined function? It doesn't really add anything, and it's not really shorter either. Compare:

```haxe
wait(1).then(tween)
waitThen(1, tween)
```

The first one seems more readable to me.